site/src/docs/components/cookie-consent/code.mdx

```diff
@@ -0,0 +1,119 @@
+  const [language, setLanguage] = React.useState('en');
+  const onLanguageChange = (newLang) => setLanguage(newLang);
+  const contentSource = {
+    siteName: 'Test site',
+    currentLanguage: language,
+    optionalCookies: {
+      cookies: [
+          {
+            commonGroup: 'statistics',
+            commonCookie: 'matomo',
+          },
+        ],
+    },
+    language: {
+      onLanguageChange,
+    },
+    focusTargetSelector: '#focused-element-after-cookie-consent-closed',
+    onAllConsentsGiven: (consents) => {
+      // called when consents are saved
+      // handle changes like:
+      if (!consents.matomo) {
+        // stop matomo tracking
+      }
+    }
+  };
+
+  const Application = () => {
+    return (
+      <div>
+        <h1 id="focused-element-after-cookie-consent-closed" tabIndex={0}>
+          Simple cookie consent example
+        </h1>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <CookieModal contentSource={contentSource} />
+      <Application />
+    </>
+  );
+};
+```
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View working modal demo in Storybook</Link>
+
+### Rendered on a page
+
+```jsx
+import { CookiePage } from 'hds-react';
+
+export const PageVersion = () => {
+  const contentSource = {
+    siteName: 'Test site',
+    currentLanguage: 'en',
+    optionalCookies: {
+      cookies: [
+        {
+          commonGroup: 'statistics',
+          commonCookie: 'matomo',
+        },
+      ],
+    },
+    onAllConsentsGiven: (consents) => {
+      // called when consents are saved
+      // handle changes like:
+      if (!consents.matomo) {
+        // stop matomo tracking
+      }
+    }
+  };
+
+  return (
+    <main>
+      <CookiePage contentSource={contentSource} />
+    </main>
+  );
+};
+```
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--page-version">View working page example in Storybook</Link>
+
+### Packages
+| Package           | Included                      | Storybook link                                                                                                                      | Source link   |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| **HDS React**     | <IconCheckCircleFill /> Yes   | <Link size="M" external href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View in Storybook</Link>  | <Link size="M" external href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/cookieconsent">View source</Link> |
+| **HDS Core**      | <IconCrossCircle /> No        | - | - |
+
+### Properties
+Note! You can find the full list of properties in the <Link size="M" href="/storybook/react/?path=/story/components-cookieconsent--default" external>React Storybook.</Link>
```

Comment:
The storybook link is broken in demo environment. Add  gatsby`withPrefix` function to href, and `openInNewTab` prop for the Link component. As the storybook goes out of the docsite app, I think we should open it into a new tab so user can more easily get back to docsite. Also not sure if we should use external here or not? It goes to a different website, but stays on the same domain.

Reply:
Fixed